.travis.yml

```diff
@@ -14,6 +14,6 @@ install:
     - python setup.py install
 script:
     - nosetests . --with-coverage
-    - flake8 --count pyglmnet --exit-zero;
+    - flake8 --count pyglmnet
```

Comment:
@jasmainak that's fine for now. just wanted to make sure developers don't feel discouraged to contribute because of flake8 norms in the future :)